data/officialLanguages.yml

```diff
@@ -281,7 +281,15 @@ PE: [es, qu, ay]
 PF: [fr]
 PG: [tpi, en, ho]
 PH: [en, fil, es, ceb, ilo, hil]
-PK: [ur, en, pa]
+# similar situation to India; only Urdu and English are official, but there are some more widely used regional languages + some differences per province/territory
+PK: [ur, en, pnb, sd, ps, skr, bal]
+PK-AJK: [ur, phr, pnb, gju, ks-Arab, hno, en]
+PK-BA: [ur, bal, ps, brh, skr, en]
+PK-GB: [ur, scl, bft, bsk, wbl, en]
+PK-KP: [ps, ur, hno, hnd, skr, khw, en]
+PK-IS: [ur, pnb, en]
+PK-PB: [pnb, ur, skr, en]
+PK-SD: [ur, sd, en]
```

Comment:
Well, as per the comment at the top of this file, only the official ones should be listed.

**Only** if the official language are only de jure official and de facto certain linguas francas / trade languages are spoken (and written!, e.g. on signs), other languages may be listed.

(If all regional languages etc. were listed in this file, it would be an absolute mess for many states in Africa. Actually, in Asia too, and South America, ...well, in a lot of places)

Reply:
OK, this is where I was confused drawing the line - very few people in Pakistan are native Urdu or English speakers, the status of both is primarily as lingua franca. So that situation is applicable here. I saw that there were some countries that had a comment saying something like no official language, or official language is trade language, etc. so that is why I put these here. I could cut these shorter and move some of them to other "other languages" file.

I am thinking maybe just keep `PK: [ur, en, pnb, sd, ps, skr, bal]` and then put the more specific ones in the other file if that makes sense.